src/options/set.rs

```diff
@@ -79,7 +79,7 @@ pub fn set_options(
     }
     opt.navigate = opt.navigate || opt.env.navigate.is_some();
     if opt.syntax_theme.is_none() {
-        opt.syntax_theme.clone_from(&opt.env.bat_theme);
+        opt.syntax_theme = opt.env.bat_theme.take();
```

Comment:
While there might be an efficiency saving here, it makes the data inconsistent, so I think we should only make changes like that for demonstrated performance wins.